node/api/apistruct/struct.go

```diff
@@ -0,0 +1,26 @@
+package apistruct
+
+import (
+	"context"
+
+	lotusstruct "github.com/filecoin-project/lotus/api/apistruct"
+	logging "github.com/ipfs/go-log/v2"
+
+	"github.com/filecoin-project/sentinel-visor/node/api"
+)
+
+var log = logging.Logger("lily-api")
+
+type LilyNodeStruct struct {
```

Comment:
Since this is the start of the new JSON-RPC API that visor will expose to allow the daemon to be controlled we should name it more concretely, e.g. just API. 

Would be nice if we could stop using the word `node` in other places where we mean `lens.Interface` and we should try to stop using it here too.

Reply:
Dropped the "node" word and moved this all to the lens package